policyengine_us/variables/gov/states/mt/tax/income/deductions/standard/mt_standard_deduction.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class mt_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana standard deduction"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
```

Comment:
need to add condition that the filer does not itemizes, use the variable: tax_unit_itemizes